app/forms/idv/in_person/address_form.rb

```diff
@@ -23,12 +23,20 @@ def submit(params)
         cleaned_errors.delete(:address1, :nontransliterable_field)
         cleaned_errors.delete(:address2, :nontransliterable_field)
 
-        FormResponse.new(
-          success: validation_success,
-          errors: cleaned_errors,
-        )
+        if validation_success
+          FormResponse.new(
+            success: validation_success,
+            errors: cleaned_errors,
+            extra: extra_analytics_attributes(params)
+            )
+
+        else
+          FormResponse.new(
+            success: validation_success,
+            errors: cleaned_errors,
+            )
```

Comment:
some goofy indentation?

```suggestion
            extra: extra_analytics_attributes(params)
          )
        else
          FormResponse.new(
            success: validation_success,
            errors: cleaned_errors,
          )
```